### ISSUE TITLE
Add search keybinding

### ISSUE THREAD

**Issue by erikzenker** [kaiwk/emacs-keybinding#1]
Usually you can search with Ctrl-s. It would be nice to have this binding in the plugin.

**Comment by kaiwk** [comment]
Yes, I have thought that before, but we can not use the original 'find in page' UI. It is possible to use [find API](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/find) to implement similar UI with HTML and CSS.  Maybe I could add this later.

 I'm not full time JS developer though, pull requests are welcome.

**Comment by MarcvdSluys** [comment]
Ctrl-S would be awesome...  Unfortunately, I'm not a JS dev either... :-(

**Comment by AstroFloyd** [comment]
Ctrl-S would be awesome...  Unfortunately, I'm not a JS dev either... :-(

**Comment by bwachter** [comment]
Proof of concept highlighting matches has been merged, I'll keep this issue open until I have jumping to matches implemented as well.

**Comment by bwachter** [comment]
I finally got annoyed enough with broken searches that I implemented DOM based search - the firefox find api I've been using for the PoC is pretty much useless.

If anybody wants to test - you should be able to pull the firefox zip from the last action run, and install that as temporary module. It requires enabling search keybindings as we'd shadow save and reload with C-s and C-r. With that setting enabled C-s or C-r will pop up a search input, highlight all matches, and further C-s or C-r will go forward or backward through the search results. Pressing RET will do an action at the highlighted search, if possible (i.e., if the search is on a link, and you press RET, it'll follow the link)

**Comment by bwachter** [comment]
This should be working now with 0.3.2,  which I've just uploaded to the firefox store
